values.yaml

```diff
@@ -89,6 +89,10 @@ concourse:
     ##
     lidarCheckerInterval:
 
+    ## Limit the number of concurrent requests to an API endpoint (Example: ListAllJobs:5)
+    ##
+    concurrentRequestLimits:
+
```

Comment:
since for now only `ListAllJobs` is allowed in the limits, would it make sense to have the chart only accept that, at least for now?

Reply:
I'm actually a bit hopeful that before concourse/concourse#5429 is merged we'll add support for more endpoints... so I've made it a draft PR again.